info/guide/guide-chapter-3.md

```diff
@@ -250,53 +250,64 @@ arg.map { value ->
     foo(value)
 }
 ```
-    
+
 **Exceptions**:
 1.	Argument lists: \
 a) Eight spaces are used to indent argument lists (both in declarations and at call sites). \
-b) Arguments in argument lists can be aligned if they are on different lines. 
+b) Arguments in argument lists can be aligned if they are on different lines.
 
 2.	Eight spaces are used if there is a newline after any binary operator.
 
 3.	Eight spaces are used for functional-like styles when the newline is placed before the dot.
 
 4.	Supertype lists: \
 a) Four spaces are used if the colon before the supertype list is on a new line. \
-b) Four spaces are used before each supertype, and eight spaces are used if the colon is on a new line. 
+b) Four spaces are used before each supertype, and eight spaces are used if the colon is on a new line.
 
-**Note:** there should be an indentation after all statements such as `if`, `for`, etc. However, according to this code style, such statements require braces. 
+**Note:** there should be an indentation after all statements such as `if`, `for`, etc. However, according to this code style, such statements require braces.
 
 ```kotlin
 if (condition)
     foo()
 ```
 
-**Exceptions**: 
+**Exceptions**:
 - When breaking the parameter list of a method/class constructor, it can be aligned with `8 spaces`. A parameter that was moved to a new line can be on the same level as the previous argument:
-    
+
 ```kotlin
 fun visit(
         node: ASTNode,
         autoCorrect: Boolean,
         params: KtLint.ExperimentalParams,
         emit: (offset: Int, errorMessage: String, canBeAutoCorrected: Boolean) -> Unit
-) {   
+) {
 }
 ```
-    
+
 - Such operators as `+`/`-`/`*` can be indented with `8 spaces`:
-    
+
 ```kotlin
-val abcdef = "my splitted" +
+val abcdef = "my split" +
                 " string"
 ```
-    
+
+- Opening and closing quotes in multiline string should have the same indentation:
```

Comment:
I think we need to emphasize here, that this rule applies only to strings with `trimIndent()` and `trimMargin()`, because otherwise the last line (before the closing triple-quote) may be meaningful part of string value: https://pl.kotl.in/elDJxYEoo

Reply:
Done [39224da](https://github.com/saveourtool/diktat/pull/1384/commits/39224da3b5f978bedef32554403a66973e17e52d)